dev-docs/modules/imRtdProvider.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: page_v2
+title: Intimate Merger Real time Data Provider
+display_name: Intimate Merger Real-time Data Submodule
+description: Intimate Merger Real-time Data Submodule
+page_type: module
+module_type: rtd
+module_code : imRtdProvider
+enable_download : true
+sidebarType : 1
+---
+
+# Intimate Merger Real time Data Provider
+{:.no_toc}
+
+* TOC
+{:toc}
+
+This module reads segments from [[Intimate Merger]] audience data platform and attaches them as targeting keys to bid requests.
```

Comment:
Not sure if [[Intimate Merger]] should be a link or a reference to something.

Reply:
Thank you for your review.
add link.